cmsplugin_filer_file/cms_plugins.py

```diff
@@ -11,6 +11,7 @@ class FilerFilePlugin(CMSPluginBase):
     render_template = "cmsplugin_filer_file/file.html"
     text_enabled = True
     admin_preview = False
+    raw_id_fields = ('file', )
```

Comment:
Filer file/image/folders have custom widgets. Django 1.8 automatically adds buttons(icons) to add/create/delete related objects near the rendered widget. Since filer widgets already have such buttons with custom functionality, Django related buttons are not needed. In order to make the Django not render these icons, we have to make them be displayed as raw_id_fields.